main.js

```diff
@@ -11,13 +11,18 @@ const BOARD_SIZE = 14;
 const HALF_BOARD_SIZE = BOARD_SIZE / 2;
 const SPHERE_RADIUS = 0.4;
 const SPHERE_WIDTH_SEGMENT = 32;
-const SPHERE_HEIGHT_SEGMENT = SPHERE_WIDTH_SEGMENT / 2;
+const SPHERE_HEIGHT_SEGMENT = SPHERE_WIDTH_SEGMENT / 3;
+const WINNING_NUMBER_OF_STONES = 5;
 
 const scene = new THREE.Scene();
 let currentPlayer = 'black';
+let currentPlayerLabel = null;
+let winnerAnnouncement = null;
+let restartButtonLabel = null;
+let stonesInScene = [];
```

Comment:
Nothing wrong with this global state, especially at this level of complexity, but often what you'll see is some namespacing to keep certain pieces of global state together, i.e.

```suggestion
const GAME_STATE = {
  let currentPlayer : 'black',
  let currentPlayerLabel : null,
  let winnerAnnouncement: null,
  let restartButtonLabel : null,
  let stonesInScene : []
}
```

This comes in handy later if you want to implement "undo" or "history" type functionality, as you can save all the different stages of the game state in an array, for example.